Code/SimDivPickers/LeaderPicker.h

```diff
@@ -329,6 +330,9 @@ struct LeaderPickerState {
     v.resize(count);
     for (unsigned int i = 0; i < count; i++) v[i] = i;
     left = count;
+    threshold = 0.0;
```

Comment:
Not a big deal, but these would be better in the initializer list.

Reply:
yep